samples/shutdown_host.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+"""
+vSphere Python SDK program for listing all ESXi datastores and their
+associated devices
+"""
+import argparse
+import atexit
+
+from pyVim import connect
+from pyVmomi import vmodl
+from pyVmomi import vim
+
+from tools import cli
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description='Process args for retrieving all the Virtual Machines')
+    parser.add_argument('-s', '--host', required=True, action='store',
+                        help='Remote host to connect to')
+    parser.add_argument('-o', '--port', type=int, default=443, action='store',
+                        help='Port to connect on')
+    parser.add_argument('-u', '--user', required=True, action='store',
+                        help='User name to use when connecting to host')
+    parser.add_argument('-p', '--password', required=True, action='store',
+                        help='Password to use when connecting to host')
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    """
+   Simple command-line program for listing all ESXi datastores and their
+   associated devices
```

Comment:
You're right :)

On Thu, Jun 11, 2015 at 6:06 PM, Michael Rice <notifications@github.com>
wrote:

> In samples/shutdown_host.py
> <https://github.com/vmware/pyvmomi-community-samples/pull/188#discussion_r32268284>
> :
>
> > +    parser.add_argument('-s', '--host', required=True, action='store',
> > +                        help='Remote host to connect to')
> > +    parser.add_argument('-o', '--port', type=int, default=443, action='store',
> > +                        help='Port to connect on')
> > +    parser.add_argument('-u', '--user', required=True, action='store',
> > +                        help='User name to use when connecting to host')
> > +    parser.add_argument('-p', '--password', required=True, action='store',
> > +                        help='Password to use when connecting to host')
> > +    args = parser.parse_args()
> > +    return args
> > +
> > +
> > +def main():
> > +    """
> > +   Simple command-line program for listing all ESXi datastores and their
> > +   associated devices
>
> You might want to update this :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vmware/pyvmomi-community-samples/pull/188/files#r32268284>
> .
>



-- 
Luis San Martin R. User #333411 linuxcounter.net